scanners/ssh-scan/parser/parser.js

```diff
@@ -180,7 +180,7 @@ async function parse(fileContent) {
         osi_layer: "APPLICATION",
         severity: "INFORMATIONAL",
         reference: {},
-        hint: "",
+        mitigation: "",
```

Comment:
It would be nicer if these could always be null when not set instead of an empty string